lib/web_ui/lib/src/engine/window.dart

```diff
@@ -195,11 +211,16 @@ class EngineFlutterWindow extends ui.SingletonFlutterWindow {
     }
   }
 
-  void computeOnScreenKeyboardInsets() {
+  void computeOnScreenKeyboardInsets(bool isEditingOnMobile) {
     double windowInnerHeight;
     final html.VisualViewport? viewport = html.window.visualViewport;
     if (viewport != null) {
-      windowInnerHeight = viewport.height!.toDouble() * devicePixelRatio;
+      if (browserEngine == BrowserEngine.webkit &&
+          operatingSystem == OperatingSystem.iOs && isEditingOnMobile) {
```

Comment:
ios => webkit, so we can just check for ios

Reply:
Done